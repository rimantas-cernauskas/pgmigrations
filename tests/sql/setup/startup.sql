/*
    ================================
    Setup test roles and permissions
    ================================
*/

CREATE EXTENSION pgtap;

CREATE ROLE __pgmigrations_superuser  WITH LOGIN SUPERUSER;
CREATE ROLE __pgmigrations_testuser   WITH LOGIN NOSUPERUSER;

-- setup permissions
SELECT pgmigrations.grant_privileges( '__pgmigrations_superuser' );
SELECT pgmigrations.grant_privileges( '__pgmigrations_testuser' );
