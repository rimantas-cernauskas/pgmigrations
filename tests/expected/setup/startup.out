/*
    ================================
    Setup test roles and permissions
    ================================
*/
CREATE EXTENSION pgtap;
CREATE ROLE __pgmigrations_superuser  WITH LOGIN SUPERUSER;
CREATE ROLE __pgmigrations_testuser   WITH LOGIN NOSUPERUSER;
-- setup permissions
SELECT pgmigrations.grant_privileges( '__pgmigrations_superuser' );
 grant_privileges 
------------------
 t
(1 row)

SELECT pgmigrations.grant_privileges( '__pgmigrations_testuser' );
 grant_privileges 
------------------
 t
(1 row)

